File: 02._data_formtas/homework/python/csvReader.py
import csv
with open("../data/car.csv") as file:

  data = csv.DictReader(file)

  carObject = [
        {
            "brand": row["brand"],
            "price": row["price"],
            "availableColors": row["availableColors"].split(","),
        }
        for row in data
    ]
      
  print('CSV FILE')
  print(carObject)

File: 02._data_formtas/homework/node/csvReader.js
import * as fs from "fs";
import Papa from "papaparse";

fs.readFile("../data/car.csv", "utf-8", (err, data) => {
  if (err) {
    throw err;
  } else {
    const parsedCsv = Papa.parse(data, { headers: false });

    const carObject = {};

    parsedCsv.data[0].map((key, index) => {
      carObject[key] = parsedCsv.data[1][index];
    });

    console.log("CSV FILE");
    console.log(carObject);
  }
});
